znai-reactjs/src/screens/landing/Landing.tsx

```diff
@@ -215,3 +150,57 @@ function groupByCategory(documentations: Documentation[]) {
     documentations: groups.get(category)!.sort(),
   }));
 }
+
+export function Landing({ documentations, type, title }: LandingProps) {
+  const [filterText, setFilterText] = useState("");
+  const [tocCollapsed, setTocCollapsed] = useState(false);
+  const landingDocMeta = useMemo(() => ({ title, type }), [type, title]);
+  const filteredDocumentations = useMemo(
```

Comment:
Not sure we actually need memos here. I could be totally wrong. The way I think about it is it is a top level component that won't have its properties changed.

Reply:
I think landingDocMeta doesn't need it (since title/type won't ever change), but the others should because they directly or indirectly rely on filterText which can change